app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileGroup.kt

```diff
@@ -61,23 +69,49 @@ fun OtherUserProfileGroup(
 }
 
 @Composable
-private fun UserGroupInformation(
-    title: String? = null,
+private fun UserGroupDetailsInformation(
+    title: AnnotatedString,
+    isSelfAdmin: Boolean,
+    onRemoveFromConversation: () -> Unit,
+) {
+    SurfaceBackgroundWrapper {
+        Column(modifier = Modifier.padding(horizontal = dimensions().spacing16x)) {
+            Height16x()
+            Text(
+                style = MaterialTheme.wireTypography.body01,
+                color = MaterialTheme.wireColorScheme.labelText,
+                text = title,
+            )
+            VerticalSpace.x16()
+            if (isSelfAdmin) {
+                WireButton(
+                    text = stringResource(id = R.string.user_profile_group_remove_button),
+                    minHeight = dimensions().spacing32x,
+                    fillMaxWidth = false,
+                    onClick = onRemoveFromConversation,
+                )
+                Height16x()
```

Comment:
You are mixing both approaches: `VerticalSpace.x16() in line 85, `Height16x()` in line 79 and 93 🤔

Reply:
removed